presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -64,31 +68,34 @@ public final void destroy()
     public void testDescribeTable()
             throws Exception
     {
-        // we need specific implementation of this tests due to specific Presto<->Mysql varchar length mapping.
-        MaterializedResult actualColumns = computeActual("DESC ORDERS").toJdbcTypes();
-
-        // some connectors don't support dates, and some do not support parametrized varchars, so we check multiple options
+        execute("CREATE TABLE tpch.test (test_bigint bigint, test_varchar varchar(255), test_vardiff varchar(15), test_double double, test_date date, test_integer integer, test_timestamp timestamp)");
+        MaterializedResult actualColumns = computeActual("DESC test").toJdbcTypes();
         MaterializedResult expectedColumns = MaterializedResult.resultBuilder(getQueryRunner().getDefaultSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
-                .row("orderkey", "bigint", "", "")
-                .row("custkey", "bigint", "", "")
-                .row("orderstatus", "varchar(255)", "", "")
-                .row("totalprice", "double", "", "")
-                .row("orderdate", "date", "", "")
-                .row("orderpriority", "varchar(255)", "", "")
-                .row("clerk", "varchar(255)", "", "")
-                .row("shippriority", "integer", "", "")
-                .row("comment", "varchar(255)", "", "")
+                .row("test_bigint", "bigint", "", "")
+                .row("test_varchar", "varchar(255)", "", "")
+                .row("test_vardiff", "varchar(15)", "", "")
+                .row("test_double", "double", "", "")
+                .row("test_date", "date", "", "")
+                .row("test_integer", "integer", "", "")
+                .row("test_timestamp", "timestamp", "", "")
                 .build();
         assertEquals(actualColumns, expectedColumns);
+        assertUpdate("DROP TABLE test");
     }
 
     @Test
     public void testInsert()
             throws Exception
     {
-        execute("CREATE TABLE tpch.test_insert (x bigint, y varchar(100))");
-        assertUpdate("INSERT INTO test_insert VALUES (123, 'test')", 1);
-        assertQuery("SELECT * FROM test_insert", "SELECT 123 x, 'test' y");
+        execute("CREATE TABLE tpch.test_insert (x bigint, y varchar(100), t_ts timestamp)");
```

Comment:
Describe test could be done for all types, while the insertion has to wait for later as some data types still have issues.